### PR TITLE
Cheat Sheet for the MacVim VimBox configuration

### PR DESCRIPTION
**Pull request by Robert-M-Muench**
See: https://github.com/jordwalke/vimbox

### CODE REVIEW

File: cheatsheets/vimbox.rb [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  # resources 'resources_dir'  # An optional resources folder which can contain images or anything else
+
+  introduction 'Modern MacVim Configuration: Mac Keyboard mappings, autocomplete, bundles etc.'  # Optional, can contain Markdown or HTML
+
+  # A cheat sheet must consist of categories
+  category do
+    id 'Mac Key Commands'  # Must be unique and is used as title of the category
+
+    entry do
+      command 'CMD+p'         # Optional
```
Comment (Kapeli) [inline cheatsheets/vimbox.rb]: All keys should be uppercase (`CMD+P` instead of `CMD+p`).

File: cheatsheets/vimbox.rb [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      command 'CTRL+p'        # Multiple commands are supported
+      name 'Open Anything and begin searching for file'    # A short name, can contain Markdown or HTML
+      notes 'Some notes'      # Optional longer explanation, can contain Markdown or HTML
+    end
+    entry do
+      command 'CMD+n'
+      name 'New Tab'
```
Comment (Kapeli) [inline cheatsheets/vimbox.rb]: Common hotkeys shouldn't be included.

File: cheatsheets/vimbox.rb [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    end
+
+    entry do
+      command 'CTRL+c'
+      name 'Kill slow search'
+    end
+  end
+
+  category do
+    id 'Javascript Support'
```
Comment (Kapeli) [inline cheatsheets/vimbox.rb]: An empty category?

**Comment by Kapeli** [comment]
Please clean up the cheat sheet and reopen the pull request. There are quite a bit of issues.

File: cheatsheets/vimbox.rb (hunk 2 above)
Comment (Robert-M-Muench) [inline cheatsheets/vimbox.rb]: The "New Tab" is the action not the hotkey.

File: cheatsheets/vimbox.rb (hunk 3 above)
Comment (Robert-M-Muench) [inline cheatsheets/vimbox.rb]: That was intended as a "todo reminder" / motivation for others to pick-it-up.

**Pull request closed without merging**